### PR TITLE
Add DTS support on Renoir

### PR DESCRIPTION
**Pull request by joechengxperi**
Add DTS post-processing support on AMD Renoir platform

### CODE REVIEW

File: tools/topology/topology1/sof/pipe-dts-codec-playback.m4 [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#
+
+ifdef(`CA_SCHEDULE_CORE',`', `define(`CA_SCHEDULE_CORE', `SCHEDULE_CORE')')
+
+# Host "Playback with codec adapter" PCM
+# with DAI_PERIODS sink and 0 source periods
+W_PCM_PLAYBACK(PCM_ID, Passthrough Playback, DAI_PERIODS, 0, SCHEDULE_CORE)
+
+W_CODEC_ADAPTER(0, PIPELINE_FORMAT, DAI_PERIODS, DAI_PERIODS, CA_SCHEDULE_CORE,
+        LIST(`          ', "CA_SETUP_CONTROLBYTES_NAME_PIPE", "CA_RUNTIME_CONTROLBYTES_NAME_PIPE"))
```
Comment (johnylin76) [inline tools/topology/topology1/sof/pipe-dts-codec-playback.m4:37]: Just a reminder that module_adapter on main had unified the kcontrol. There is no "setup" and "runtime" config anymore.
The current code won't cause problems anyway.

File: src/arch/xtensa/configs/override/renoir_chrome_dts.config [hunk 2]
```diff
@@ -0,0 +1,12 @@
+CONFIG_RENOIR=y
+CONFIG_TRACE_CHANNEL=7
+CONFIG_WRAP_ACTUAL_POSITION=y
+CONFIG_AMD_BT=n
+CONFIG_CORE_COUNT=1
+CONFIG_LP_MEMORY_BANKS=0
+CONFIG_HP_MEMORY_BANKS=10
+CONFIG_FORMAT_CONVERT_HIFI3=n
+CONFIG_LP_SRAM=n
+CONFIG_HAVE_AGENT=n
```
Comment (johnylin76) [inline src/arch/xtensa/configs/override/renoir_chrome_dts.config:10]: The configs which are already in src/arch/xtensa/configs/renoir_defconfig doesn't need to be in the override config (line1 to line10). Unless the config value is different.

File: src/audio/module_adapter/module/dts.c [hunk 3]
```diff
@@ -316,7 +316,7 @@ static int dts_codec_apply_config(struct processing_module *mod)
 		/* If param->size is less than param_header_size, then this param is not valid */
 		if (param->size < param_header_size) {
 			comp_err(dev, "dts_codec_apply_config() param is invalid");
-			return -EINVAL;
+			return 0;
```
Comment (johnylin76) [inline src/audio/module_adapter/module/dts.c:319]: I don't remember about this. What is this for?

File: src/arch/xtensa/configs/override/renoir_chrome_dts.config (hunk 2 above)
Comment (joechengxperi) [reply src/arch/xtensa/configs/override/renoir_chrome_dts.config]: Remove the duplicated lines in the override config.

File: src/audio/module_adapter/module/dts.c (hunk 3 above)
Comment (joechengxperi) [reply src/audio/module_adapter/module/dts.c:319]: DTS can't be loaded successfully without this change.

Comment (johnylin76) [reply src/audio/module_adapter/module/dts.c:319]: ACK

**Comment by johnylin76** [comment]
Looks good to me.
@lgirdwood could you help with the approval and merge. The change in this PR should be component-specific.

**Review by lgirdwood — APPROVED** [review]

**Review by johnylin76 — APPROVED** [review]
LGTM

**Comment by johnylin76** [comment]
Just found I forgot to click the approval...
Hi @lgirdwood could you help to merge this PR? Thanks.
(There is "On-device Test Failed:TGLU_UP_HDA_ZEPHYR" but I don't think it's relevant)

**Pull request merged**